docs/api-docs-theme/layouts/default.hbs

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html class="default no-js">
+<head>
```

Comment:
is this file copied from somewhere else, or was it generated?

Reply:
Copied from the default theme per typedoc docs.